test/powershell/install_modules_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+# Import test library for `check` command
+source dev-container-features-test-lib
+
+# Extension-specific tests
+check "az.resources" pwsh -Command "(Get-Module -ListAvailable -Name Az.Resources).Version.ToString()"
```

Comment:
Can you write a test to validate that the version installed for `az.resources` is `2.5.0`?